catalyst_voices/pubspec.yaml

```diff
@@ -32,11 +32,14 @@ dependencies:
   catalyst_voices_view_models:
     path: ./packages/catalyst_voices_view_models
   collection: ^1.18.0
+  dotted_border: ^2.1.0
```

Comment:
Please add to melos.yaml if not added already. Please check all new dependencies